src/components/composer/composer/composer.ts

```diff
@@ -811,15 +781,22 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
     this.autoCompleteState.values = values.map((value) => ({ text: value, description: "" }));
     this.autoCompleteState.getHtmlContent = (value) => [{ value }];
   }
+
+  get popoverProps(): PopoverProps {
+    return {
+      anchorRect: this.props.getContainerRect?.() || getBoundingRectAsPOJO(this.composerRef.el!),
```

Comment:
do we really need the props `props.getContainerRect` ? Can't we always use the rect of `this.composerRef.el` ?
It's slightly "décallé" but it's not choquant. It can be seen as a design choice

Reply:
I think it's a bit uglier but sure